tools/samtool_filter2/samtool_filter2.xml

```diff
@@ -115,6 +115,7 @@ samtools view -o "$output1" $header
     <param name="read_group" type="text" value="" label="Select alignments from Read Group"
            help="(-r) Requires headers in the input SAM or BAM, otherwise no alignments will be output"/>
     <param name="bed_file" type="data" format="bed" optional="true" label="Output alignments overlapping the regions in the BED FILE" help="(-L)"/>
+    <param name="pos_or_neg_select" checked="false" label="Select everything but the following:" type="boolean" truevalue="-U" falsevalue="-o" help="" />
```

Comment:
`-o` needs to stay even if you want to have -U, correct? So just use falsevalue=""
I would call this option "Enable inverse selection" of something like that, the current naming sounds wired because it is referring to "following" which can be a lot.